README.md

```diff
@@ -1,7 +1,15 @@
-# Community-events
+# Community Events @ 🤗
 
-This repository is meant for all community events organized by 🤗 HuggingFace.
+A central repository for all community events organized by 🤗 HuggingFace. Come one, come all!
+We're constantly finding ways to democratise the use of ML across modalities and languages. This repo contains information about all past, present and upcoming events.
 
-## Content
+## Hugging Events
 
-May 16 - May 31: [Blocks Party 🎊](gradio-blocks)
+| **Event Name**                                                          | **Dates**       | **Status**                                                                                                   |
```

Comment:
Maybe add the year in the dates column noo

Reply:
Good idea.. doing that right now.